Source/d_deh.c

```diff
@@ -1086,6 +1098,28 @@ struct {
   {"TRANSLUCENT",  0x80000000}, // apply translucency to sprite (BOOM)
 };
 
+#define DEH_MOBJFLAGMAX_MBF21 (sizeof(deh_mobjflags_mbf21) / sizeof(*deh_mobjflags_mbf21))
+
+static const struct deh_flag_s deh_mobjflags_mbf21[] = {
```

Comment:
MBF21 added new mobj flags, so we need to add a new field to mobj_t and broke saves. Maybe we should introduce some kind of versioning?